frontend/src/redux/reducers/popupLocationReducer.tsx

```diff
@@ -28,6 +28,8 @@ export const popupLocationReducer = (
       return { ...state, isLoading: true };
     case LocationActionTypes.LOADING_END:
       return { ...state, isLoading: false };
+    case LocationActionTypes.ADD_COMMENT:
+      return { ...state, comments: action.payload };
```

Comment:
actually, this is bad practice to store loading state and data in the same object...

has to be reworked in next sprint!